projects/laji/src/app/shared/service/laji-api.interceptor.ts

```diff
@@ -87,14 +86,14 @@ export class LajiApiInterceptor implements HttpInterceptor {
 
     setTimeout(() => {
       if (request.body instanceof FormData) {
-        const data = [];
+        const data: any[] = [];
```

Comment:
`const data: Observable<IFileLoad>[] = [];`
(`IFileLoad` also needs to be exported)